homeassistant/components/switch/wemo.py

```diff
@@ -24,58 +23,20 @@
 MAKER_SWITCH_TOGGLE = "toggle"
 
 
-def _find_manual_wemos(pywemo, static_config):
-    for address in static_config:
-        port = pywemo.ouimeaux_device.probe_wemo(address)
-        if not port:
-            _LOGGER.warning('Unable to probe wemo at %s', address)
-            continue
-        _LOGGER.info('Adding static wemo at %s:%i', address, port)
-        url = 'http://%s:%i/setup.xml' % (address, port)
-        yield pywemo.discovery.device_from_description(url, None)
-
-
 # pylint: disable=unused-argument, too-many-function-args
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Find and return WeMo switches. """
-    import pywemo
     import pywemo.discovery as discovery
 
```

Comment:
Perhaps worth checking that SUBSCRIPTION_REGISTRY has been initialised, and give an error and return false so that if someone tried to load the switch platform without the wemo component - it fails simple.